app/screens/home/search/search.tsx

```diff
@@ -137,8 +138,7 @@ const SearchScreen = ({teamId}: Props) => {
             <NavigationHeader
                 isLargeTitle={true}
                 onBackPress={() => {
-                    // eslint-disable-next-line no-console
-                    console.log('BACK');
+                    logInfo('BACK');
                 }}
```

Comment:
this entire prop can be removed as it was there initially for testing/example purposes